contents/blog/launch-week-universe-of-new-features.md

```diff
@@ -0,0 +1,78 @@
+date: 2022-03-22
+title: PostHog Launch Week I: A Universe of New Features
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ..
+featuredImageType: full
+author: ["andy-vandervell"]
+categories: ["Product updates"]
+---
+
+PostHog made huge strides in 2021. To name just a few landmarks, we: 
+
+- Raised a [$15 million Series B ahead of schedule](/blog/why-we-raised-a-15m-series-b-ahead-of-schedule)
+- Became one of [YC's top-valued companies](/blog/yc-top-companies)
+- Rebranded our website [in just four weeks](/blog/postmortem-rebrand)
+- Migrated from [postgresql to ClickHouse](/blog/how-we-turned-clickhouse-into-our-eventmansion)
+- Achieved product market fit for PostHog Scale
+
+All of this (and more) lead to huge growth. Users grew by 4x, the number of open source contributors doubled, we acquired numerous reference customers, and went from zero revenue to significant revenue.
+
+As an open-source product, we're also delighted to see huge growth in our Slack community - now at over 2,000 members - and GitHub stars, which now stand at over 7,000.
+
+But this is just the beginning. We've also been busy making PostHog broader, more comprehensive, and a more powerful tool for making great products. 
+
+To celebrate, starting on Monday, March 21 we'll be officially launching a new feature or initative every day for a week.
+
+This is... PostHog Launch Week I: A Universe of New Features.
+
+## Chapter I: Experimentation
+
+- **Releasing:** Monday, March 21
+- **Directed by:** xxxx
+- **Starring:** Neil Kakkar
+- **Introducing:** A/B testing, multivariate tests and statistical significance
+
+Critics are calling it "the most statistically signicant new feature since [a joke Engineers will get, help please!]"
```

Comment:
```suggestion
Critics are calling it "Neil's most statistically significant new feature since [Correlation Analysis](/docs/user-guides/correlation)"
```

Reply:
Not a joke, but it works. Thanks.

Reply:
I refuse to pander to Joe, but that was genuinely quite funny